test/testvfdswmr.sh.in

```diff
@@ -1047,48 +1047,48 @@ fi
 #
 BIGSET_n=25                         # -n option: # of iterations
 BIGSET_few_s=10                     # -s option: # of datasets (for few_big test)
-BIGSET_many_s=100                   # -s option: # of datasets (for many_small test)
+BIGSET_many_s=50                    # -s option: # of datasets (for many_small test)
 #
 #
 # Setting for exhaustive and quick runs
 #
 if [[ "$HDF5TestExpress" -eq 0 ]] ; then         # exhaustive run
-    BIGSET_n=200
-    BIGSET_few_s=100
-    BIGSET_many_s=1000
+    BIGSET_n=100
+    BIGSET_few_s=25
+    BIGSET_many_s=100
 elif [[ "$HDF5TestExpress" -gt 1 ]]; then        # quick run
     BIGSET_n=10
     BIGSET_few_s=3
-    BIGSET_many_s=50
+    BIGSET_many_s=25
 fi
 #
 #
-for options in "-d 1" "-d 1 -F" "-d 2" "-d 2 -F" "-d 1 -t" "-d 1 -t -F" "-d 1 -t -R" "-d 1 -V" "-d 1 -M" "-d 1 -V -F" "-d 1 -M -F"; do
+for options in "-d 1" "-d 1 -F" "-d 2 -l 16" "-d 2 -F -l 16" "-d 1 -t" "-d 1 -t -F" "-d 1 -t -R" "-d 1 -V" "-d 1 -M" "-d 1 -V -F" "-d 1 -M -F"; do
 	if [ ${do_many_small:-no} = no ]; then
 		continue
 	fi
 	#
-	# Test many small datasets of one and two dimensions.
+	# Test many small datasets of two or three dimensions.
 	#
 	# Perform 25 iterations on 100 extensible datasets configured with
-	# 2D 16x16 chunks or 3D 8x16x16 chunks of 32-bit unsigned integer elements,
+	# 2D 16x16 chunks or 3D 1x16x16 chunks of 32-bit unsigned integer elements,
 	# expanding each dataset by a chunk in one dimension (up to 25x1
 	# 16x16 chunks) on each iteration.
 	#
 	# Perform the test again, extending each dataset
 	# in *two* dimensions (up to 25x25 16x16 chunks).
 	#
         # If testing 3D datasets (-t option), extending each dataset along the
-        # first dimension (up to 25 8x16x16)
+        # first dimension (up to 25 1x16x16)
         #
 
 	echo launch vfd_swmr_bigset_writer many small, options $options
 	catch_out_err_and_rc vfd_swmr_bigset_writer \
-	    ../vfd_swmr_bigset_writer -n $BIGSET_n $options -s $BIGSET_many_s -e 8 -r 16 -c 16 -q -l 3 &
+	    ../vfd_swmr_bigset_writer -n $BIGSET_n $options -s $BIGSET_many_s -e 1 -r 16 -c 16 -q &
```

Comment:
In the future, we should specify the path to the executables in one spot. It makes it easier to change.